python/coinbase-agentkit/coinbase_agentkit/action_providers/aave/constants.py

```diff
@@ -0,0 +1,125 @@
+"""Constants for Aave action provider."""
+
+SUPPORTED_NETWORKS = ["base-mainnet"]
+
+# Asset addresses for supported networks
+ASSET_ADDRESSES = {
+    "base-mainnet": {
+        "weth": "0x4200000000000000000000000000000000000006",
+        "usdc": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
+        "cbeth": "0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22",
+        "wstETH": "0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452",
+        "cbBTC": "0xcbb7C0000aB88B473b1f5aFd9ef808440eed33Bf",
+        "GHO": "0x6Bb7a212291678b212391A10eE80a28B6f28C568",
```

Comment:
please fix wrong GHO address
```suggestion
        "GHO": "0x6Bb7a212910682DCFdbd5BCBb3e28FB4E8da10Ee",
```